docs/src/operate/security/http_local_authentication.md

```diff
@@ -0,0 +1,405 @@
+---
+title: HTTPS Local Access
+tags: [Operate, Security, HTTP]
+sidebar_position: 6
+---
+
+Thin-edge provides two services over HTTP:
+- The [File Transfer Service](../../references/tedge-file-transfer-service.md) is used by the mappers and the child devices to transfer files locally.
+- The [Cumulocity Auth Proxy](../../references/tedge-cumulocity-proxy.md) acts as a local proxy to the Cumulocity IoT REST API.
```

Comment:
Internally, we've referred to this as the auth-proxy to disambiguate it with the `c8y-proxy` crate we have. In the documentation page for this, I specifically avoided calling it "auth proxy", because that ambiguity doesn't exist in the public API. So I think we should probably call it "Cumulocity Proxy" here (or if we want to say "Auth" here, we should say "Authentication" instead)
```suggestion
- The [Cumulocity Proxy](../../references/tedge-cumulocity-proxy.md) acts as a local proxy to the Cumulocity IoT REST API.
```

Reply:
Fixed